tests/test-ray-tracing-common.h

```diff
@@ -0,0 +1,734 @@
+#pragma once
+
+#include "testing.h"
+#include <slang-rhi/acceleration-structure-utils.h>
+#include <vector>
+#include <string>
+
+// Mosts ray tracing tests only need:
+// - a BLAS with simple geometry
+// - a TLAS with a single instance
+// - a pipeline
+// - a straightforward shader table
+//
+// This header provides classes and functions that provide these common building blocks.
+
+namespace rhi::testing {
+struct Vertex
+{
+    float position[3];
+};
+
+struct TriangleBlas
```

Comment:
minor naming conventions stuff (no need to change):
- for class names we should probably use `BLAS` and `TLAS`, so `TriangleBLAS` here
- for members and variables it would be `blas` and `tlas`, so `blasBuffer` instead of `BLASBuffer`